### ISSUE TITLE
Rule Change: multiline-ternary add always-start option

### ISSUE THREAD

**Issue by caub** [eslint/eslint#15934]
### What rule do you want to change?

multiline-ternary

### What change to do you want to make?

Implement suggestions

### How do you think the change should be implemented?

A new option: `always-start`

### Example code

```js
foo > bar
    ? value1
    : value2;
```

### What does the rule currently do for this code?

It rejects it

### What will the rule do after it's changed?

It would allow it with `"multiline-ternary": ["error", "always-start"]`

### Participation

- [X] I am willing to submit a pull request to implement this change.

### Additional comments

This is also how typescript lsp formats ternaries

I really like this style, particularly with React, it make things more obvious that we are in a ternary

**Comment by mdjermanovic** [comment]
Hi @caub, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates), so we are not adding new options to the `multiline-ternary` rule. You can always copy the existing rule into a custom rule and make it behave the way you want.
